components/art/NodeGrid.vue

```diff
@@ -2,7 +2,7 @@
   <svg
     ref="nodeGrid"
     :width="width"
-    viewBox="0 0 615.3 595.2"
+    viewBox="0 0 715.3 595.2"
```

Comment:
Increased viewbox value to see full spheres at the end, spheres were getting cropped